loadouts/cargo-list.sqf

```diff
@@ -127,6 +127,15 @@ switch (_x) do {
 			["R3F_FUM40",  40]
 		] call BRM_FMK_fnc_addItems;
 	};
+	
+	case "SMAW":{
```

Comment:
Instead add to "at", if specAT is a SMAW, so old missions will also get the items.